.github/workflows/ci.yml

```diff
@@ -69,19 +65,17 @@ jobs:
         run: composer install --prefer-dist --no-progress
 
       - name: Run phpstan
-        run: make phpstan
+        run: just lint
 
   tests:
-    name: Tests
-
     runs-on: ubuntu-latest
 
     strategy:
       fail-fast: false
       matrix:
-        env:
-          - AUTOLOAD=0
-          - AUTOLOAD=1
+        autoload:
```

Comment:
because `just` recipes can take arguments, we can replace the env-looking value with the actual value we want and pass it in directly